### ISSUE TITLE
Submitting form

### ISSUE THREAD

**Issue by web-dept** [formtools/core#283]
I have an external form that I am submitting direct. When I submit the external form, the submission is registered in the database but the recipient doesn't receive the email. 

When I run the test from Form Tools, I receive the test emails if I select HTML or Text. If I select HTML & Text, the test fails.

**Comment by benkeen** [comment]
Hi @web-dept, this is actually a pretty common problem with some mail clients. Sending multipart emails (both text + html) only works with some of them. To make it work more consistently, I'd suggest installing the Swift Mailer module: 
https://docs.formtools.org/modules/swift_mailer/

That irons out all the odd wrinkles with sending emails.

**Comment by web-dept** [comment]
Thank you. Does Swift Mailer work with Gmail SMTP servers?

**Comment by wolftone1916** [comment]
Just in case it's the same issue I ran into in setting up the email. Servers like yahoo for security reasons won't allow email to come in that has the same return email as the one receiving it. So like if you are using the same email in the one who receives it and the one who sent it, the email server won't likely accept it. I created an email account that was webmaster@mydomain.com and used that for replies and to be listed as the sender, I then was able to get emails from the form.
